File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        s = set(nums)
        maxCount = 0
        for n in s:
            count = 0
            # Check if n has a left neighbor
            if n - 1 in s:
                continue
            # Right neighbors
            count += 1
            x = n + 1
            while x in s:
                count += 1
                x += 1
            if count >= maxCount:
                maxCount = count
        return maxCount

File: Data Structures & Algorithms/valid-palindrome-ii/submission-2.java
class Solution {
    public boolean isPalindrome(String s){
        int right = s.length() - 1;
        int left = 0;

        while (left < right){
            if (s.charAt(left) != s.charAt(right)){
                return false;
            }
            left++;
            right--;
        }
        return true;
    }
    public boolean validPalindrome(String s) {
        if (isPalindrome(s)){
            return true;
        }

        int left = 0;
        int right = s.length() - 1;
        while (left < right){
            if (s.charAt(left) != s.charAt(right)){
                return isPalindrome(s.substring(0, left) + s.substring(left + 1)) || 
                       isPalindrome(s.substring(0, right) + s.substring(right + 1));            
            }
            left++;
            right--;
        }
        return true;
        
    }
}